### PR TITLE
[TM ONLY] Disables Sideplane, Makes the Game Playable

### PR DESCRIPTION
**Pull request by RimiNosha**
## About The Pull Request
Anyone who's played since december knows that sideplane was a mistake in it's current state.

Let's send it to the shadow realm, at least until botany and items are fixed.

This is an upstream issue, and I have no control over it, before I get someone commenting that I should fix it. I also do not know how sideplane works at all.

## How This Contributes To The Skyrat Roleplay Experience

I'd like to pick up my fucking items, thanks.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image
image
image

# OH LOOK, I CAN SEE MY FUCKING ITEM
</details>

## Changelog

No CL, cause the only way this gets merged is if we fork.
<details>
<summary>Obsolete stuff</summary>

Reverts:
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/18070
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/18427
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/18587
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/18590 (partially, tram signs might look slightly wonky, in order to avoid map conflicts)
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/18879
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/18930
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/19059
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/19217
- https://github.com/Skyrat-SS13/Skyrat-tg/pull/19060

Sadly the new starlight values used planes extensively, I'm not converting them.
image
</details>

### CODE REVIEW

**Comment by Iamgoofball** [comment]
botany isn't broken upstream, whatever issue you've got with botany down here is almost assuredly not the fault of SIDEMAP and is the fault of skyrat specific code bugs

**Comment by RimiNosha** [comment]
https://github.com/tgstation/tgstation/issues/72765
https://github.com/tgstation/tgstation/issues/72569

**Comment by Iamgoofball** [comment]
these are incredibly minor cosmetic issues, i assumed you meant they were completely unplayable given the fact you're reverting a complete refactor of the render stack

**Comment by GoldenAlpharex** [comment]
> these are incredibly minor cosmetic issues, i assumed you meant they were completely unplayable given the fact you're reverting a complete refactor of the render stack

It's not really minor considering it's a refactor of the render stack that made it so items and overlays start flickering like crazy, and there's like only two or three people that understand enough of it to even be able to maybe figure out a potential fix.

The main frustration that comes from things like these is that they get merged before some of the very obvious bugs get fixed, and now we're left to deal with them for months on end.

**Comment by Iamgoofball** [comment]
that's understandable and i wish we could get more progress done on this front, but please keep in mind that we functionally cannot provide effective troubleshooting on rendering bugs involving planes upstream at /tg/ if sidemap is shut off since we're operating on sidemap unless wallening explodes

**Comment by GoldenAlpharex** [comment]
> that's understandable and i wish we could get more progress done on this front, but please keep in mind that we functionally cannot provide effective troubleshooting on rendering bugs involving planes upstream at /tg/ if sidemap is shut off since we're operating on sidemap unless wallening explodes

I'm well aware, but it's also why stuff like this should have been fixed as a priority after the sidemap was introduced, if not being a reason to wait before merging the sidemap PR to begin with.

I love potato but this kind of bug is disheartening to see and/or hear about every single day.

File: code/world.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	hub_password = "kMZy3U5jJHSiBQjr"
 	name = "/tg/ Station 13"
 	fps = 20
-	map_format = SIDE_MAP
+	//map_format = SIDE_MAP
```
Comment (GoldenAlpharex) [inline code/world.dm]:
```suggestion
	//map_format = SIDE_MAP // SKYRAT EDIT - Waiting for it to be fixed and ready for production
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by Gandalf2k15** [comment]
Removing this from TM as it seems to have been fixed.

**Comment by Gandalf2k15** [comment]
This has been fixed.

**Comment by GoldenAlpharex** [comment]
It's, in fact, still not fixed, as far as we could tell, unfortunately. I also know we're likely not going to be the ones that will fix it ourselves, so I think this is still relevant.

**Comment by Gandalf2k15** [comment]
You dont need prs open to tm them

**Pull request closed without merging**